trafficlight/internals/client.py

```diff
@@ -174,10 +174,11 @@ async def accept_crosssign(self) -> None:
     async def verify_crosssign(self) -> None:
         await self._perform_action({"action": "verify_crosssign_emoji", "data": {}})
 
-    async def create_room(self, room_name: str) -> None:
-        await self._perform_action(
+    async def create_room(self, room_name: str) -> str:
+        response = await self._perform_action(
             {"action": "create_room", "data": {"name": room_name}}
         )
+        return cast(str, response["response"])
```

Comment:
I made a comment on the related PR in the adapter; i think we should have a little structure here to allow us to extend later (rather than only returning a string). If we change it in the adapter we'll need to change it here.